tests/suite/math/frac.typ

```diff
@@ -45,3 +45,19 @@ $ a_1/b_2, 1/f(x), zeta(x)/2, "foo"[|x|]/2 \
 --- math-frac-gap ---
 // Test that the gap above and below the fraction rule is correct.
 $ sqrt(n^(2/3)) $
+
```

Comment:
A few more tests, also in inline math would be nice.

Reply:
could you also add tests with explicit `frac` calls to make sure it behaves as intended with the deparenthesization logic?